main_pipeline/data_utils/__init__.py

```diff
@@ -0,0 +1 @@
+# Utilities for dataverse or whatever we decide to use for data hosting will go here. Also putting data processing utils here
```

Comment:
This file should be deleted (I forgot to in my branch as well but it's redundant now).